draft-irtf-cfrg-aead-limits.md

```diff
@@ -111,16 +118,16 @@ confidentiality and integrity.  Excessive use of the same key can give an
 attacker advantages in breaking these properties.  This document provides simple
 guidance for users of common AEAD functions about how to limit the use of keys
 in order to bound the advantage given to an attacker.  It considers limits in
-both single- and multi-user settings.
+both single- and multi-key settings.
 
 --- middle
 
 # Introduction
 
 An Authenticated Encryption with Associated Data (AEAD) algorithm
 provides confidentiality and integrity. {{!RFC5116}} specifies an AEAD
-as a function with four inputs -- secret key, nonce, plaintext,
-and optional associated data -- that produces ciphertext output and error code
+as a function with four inputs -- secret key, nonce, optional plaintext,
```

Comment:
Is it optional or just maybe zero-length?  I know that this distinction is sometimes important, but it is here?